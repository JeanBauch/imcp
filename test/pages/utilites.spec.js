import { mount } from '@vue/test-utils';

import Utilites from '~/pages/utilites.vue';

describe('Pages:index', () => {
	let mountConfig;

	beforeEach(() => {
		mountConfig = {};
	});

	describe('Snapshots', () => {
		it('Should render all child componets', () => {
			const wrapper = mount(Utilites, mountConfig);
			expect(wrapper.html()).toMatchSnapshot();
		});
	});
});
