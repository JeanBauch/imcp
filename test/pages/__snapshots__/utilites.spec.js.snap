// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages:index Snapshots Should render all child componets 1`] = `
"<div class=\\"bg-background flex flex-col items-center justify-center gap-4 md:gap-8\\">
  <h1>Utilizes testes</h1>
  <h1>Está tela ainda está sendo criada!</h1>
  <h3>outro teste</h3>
</div>"
`;
